store/storetest/user_store.go

```diff
@@ -325,6 +334,13 @@ func testGetAllUsingAuthService(t *testing.T, ss store.Store) {
 	})).(*model.User)
 	defer func() { store.Must(ss.User().PermanentDelete(u3.Id)) }()
 	store.Must(ss.Team().SaveMember(&model.TeamMember{TeamId: teamId, UserId: u3.Id}, -1))
+	store.Must(ss.Bot().Save(&model.Bot{
```

Comment:
For context, in case these changes seem "too convenient": I had already pre-flighted the necessary bot users in the commit to master that introduced the dynamic SQL generation, but hadn't marked them as bots there. These changes neatly slide back in.

Reply:
I did have to check that :P